src/main/java/frc/robot/commands/auto/PathPlannerCommand.java

```diff
@@ -1,120 +1,122 @@
-                        drive.getPathplannerXController(), // X controller can't normal PID as pathplanner has Feed Forward
-                        drive.getPathplannerYController(), // Y controller can't normal PID as pathplanner has Feed Forward
-                        drive.getPathplannerRotationController(), // Rotation controller can't normal PID as pathplanner has Feed Forward
-                        (chassisSpeeds) -> {
-                            drive.setChassisSpeeds(chassisSpeeds, false);
-                        }, // chassis Speeds consumer
-                        useAllianceColor,  // use Alliance color
-                        drive, // Requires this drive subsystem
-                        isPerpetual
-                                             )
-                   );
-    }
-
-    public static PPSwerveControllerCommand createSwerveControllerCommand(
-            PathPlannerTrajectory trajectory, Supplier<Pose2d> poseSupplier,
-            PIDController xController, PIDController yController, PIDController rotationController,
-            Consumer<ChassisSpeeds> outputChassisSpeeds, boolean useAllianceColor, DrivetrainImpl drive, boolean isPerpetual) {
-        if (isPerpetual) {
-            return new PPSwerveControllerCommandPerpetual(
-                    trajectory,
-                    poseSupplier,
-                    xController,
-                    yController,
-                    rotationController,
-                    outputChassisSpeeds,
-                    useAllianceColor,
-                    drive
-            );
-        }
-
-        return new PPSwerveControllerCommand(
-                trajectory,
-                poseSupplier,
-                xController,
-                yController,
-                rotationController,
-                outputChassisSpeeds,
-                useAllianceColor,
-                drive
-        );
-    }
-}
+//package frc.robot.commands.auto;
+//
+//import com.pathplanner.lib.PathConstraints;
+//import com.pathplanner.lib.PathPlanner;
+//import com.pathplanner.lib.PathPlannerTrajectory;
+//import com.pathplanner.lib.PathPoint;
+//import com.pathplanner.lib.commands.PPSwerveControllerCommand;
+//import edu.wpi.first.math.controller.PIDController;
+//import edu.wpi.first.math.geometry.Pose2d;
+//import edu.wpi.first.math.kinematics.ChassisSpeeds;
+//import edu.wpi.first.wpilibj.DriverStation;
+//import edu.wpi.first.wpilibj2.command.InstantCommand;
+//import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+//import frc.robot.constants.miscConstants.AutoConstants;
+//import frc.robot.subsystems.drive.DrivetrainImpl;
+//import frc.robot.util.ConversionUtils;
+//import frc.robot.util.PathGroupLoader;
+//
+//import java.util.ArrayList;
+//import java.util.List;
+//import java.util.function.Consumer;
+//import java.util.function.Supplier;
+//
+//
+//public class PathPlannerCommand extends SequentialCommandGroup {
```

Comment:
As with other commented out code, we should remove it if it is going to the main branch